src/Nancy/HttpLinkRelation.cs

```diff
@@ -0,0 +1,180 @@
+﻿namespace Nancy
+{
+    using System;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// The relation that identifies the semantics of a <see cref="HttpLink"/> contained in an RFC 5899 Link HTTP header,
+    /// as built by the <see cref="HttpLinkBuilder"/>.
+    /// </summary>
+    /// <seealso cref="System.Uri" />
+    [Serializable]
+    public class HttpLinkRelation : Uri, IEquatable<HttpLinkRelation>
```

Comment:
Well... According to RFC 5899, it **is** an URI (or an IRI, to be exact), so I thought it made sense to inherit to make that relationship obvious.